### ISSUE TITLE
Provide way to donate

### ISSUE THREAD

**Issue by L3P3** [Feodor2/Mypal#284]
I donated to Mozilla, to Pale Moon and I want to donate to you also, @Feodor2. I want to donate equal to all 3 parties since to me XP support is very very important so to me it does not matter how much lines of code come from which party. You may provide a paypal.me link or bitcoin address or whatever, I do not want to have the feeling of using your product for years without giving back. Thanks.

**Comment by Feodor2** [comment]
I live in country Ukraine with very oppressor and criminal authority, no one convienet way to transfer money. paypal is blocked webmoney and skirll too, only yandex via vpn is available.
So i think it will be better to hire a person with good programming skills at any international labour bureau, who fix couple bugs for us

**Comment by AroKol78** [comment]
> I live in country Ukraine with very oppressor and criminal authority, no one convienet way to transfer money. paypal is blocked webmoney and skirll too, only yandex via vpn is available.

many people in the west do not understand the specifications of Eastern Europe
> So i think it will be better to hire a person with good programming skills at any international labour bureau, who fix couple bugs for us

 :+1:

**Comment by L3P3** [comment]
https://dash.org/

You could try this. It should work in any country I think.
I don't know how. I know there are german companies that work with ukrainian developers and pay money officially, there must be ways.
Why hire another developer? I think you do good work so far.

**Comment by L3P3** [comment]
Or would it be possible to port your XP compatiblity back to Pale Moon? Maybe I can pay them so they make it compatible and fix bugs. Maybe it would be better on long term or? No idea.

**Comment by AroKol78** [comment]
> Maybe it would be **better on long term**

I think so too
> No idea.

eventually - postal order, postman.

**Comment by Feodor2** [comment]
>https://dash.org/ 

shall look

>pay money officially

No, authority take away bigger part then, i do not want you to feed scum.

>Why hire another developer? I think you do good work so far. 

I not skilled good enough.

>possible to port your XP compatiblity back to Pale Moon

You are to try with money then.

**Comment by andres99** [comment]
Thank you for MyPal! I am still using XP and therefore, it is great news! If you ever find a way to accept donations, I would donate.

**Comment by AroKol78** [comment]
> If you ever find a way to accept donations, I would donate.

I also (bank transfer, postal order, ... ?) - for the work done so far
MyPal is the best browser for **XP** - easy installation, languages, automatic updates, addons, low requirements
and most of all the creator's passion

For the good of the future the browser https://github.com/Feodor2/Mypal/issues/284#issuecomment-723567871

**Comment by NoneRain** [comment]
What about cryptocurrency? That could be an option...

**Comment by L3P3** [comment]
I already recommended DASH for a reason. Is technically a very good way. Very easy to use and I would donate a lot of it as soon as Feodor gives me his DASH address.

**Comment by andres99** [comment]
Well, I have nothing against one or another particular cryptocurrency but a DASH transfer over DASH network (from Binance) would at the moment cost $0.78 while a TRX transfer over TRC20 network would cost $0.14.

**Comment by L3P3** [comment]
When I donate 1000€, I do not give a shit about less than 1€, sorry. :-)

**Comment by andres99** [comment]
The easiest way is probably to register an account at a crypto exchange, for example Binance, and get the donations to a Binance wallet (in DASH, TRX, BTC, USDT or any other crypto you like). For USDT or TRX, payments over the TRC20 network are probably fastest and cheapest now. For DASH you can use the DASH network. To get the wallet address for the specific crypto, use "Deposit", select the crypto and then it shows you your wallet address for that crypto. After you have a donation on your account, you can wait until its price increases, or you can immediately exchange it into your money (USD, EUR, or more). Are you in Ukraine? I have friends in Ukraine who use Binance and they get the grivnas out of the exchange by using the P2P option.

**Comment by L3P3** [comment]
I cannot recommend DASH enough, especially for its security/anonymity. DASH is like cash, you can just ask anyone if he wants to exchange your DASH to money like you sell some tools or food. On ebay for example, or if there is some local alternative where people can offer things.

**Comment by Feodor2** [comment]
I put my dash number here right? But i do not want mine at all and did not find the option to disable mining so have closed it, will it works if closed.

**Comment by L3P3** [comment]
I would put the DASH receive number to the end of your README like
```
For donations via DASH: Xxxxxxxxxxxxxxxxxxxxxxxxxxxx
```
Normal wallets do not mine by themselves. If they do, you see CPU usage 100%.
Your receive number(s) are not secret. You can generate multiple ones and put them everywhere, no one can track that they are yours unless you use the same numbers on different places.

**Comment by L3P3** [comment]
But when I donate, you buy new pc, no more Windows XP and end Mypal? Hope not!!!)))))

**Comment by Feodor2** [comment]
so this is the right code for donation i think.
XetK1TDBqVAyqnTJUd65g9Pi1HcQEtFF2c

@L3P3 how much your going to donate then, i don't need new pc, i think about hire a programmer.

**Comment by andres99** [comment]
For the sake of clarity, you should indicate if that address is for DASH or any other cryptocurrency. Some donators may recognise DASH addresses and know the system well – while others may not.

**Comment by L3P3** [comment]
@andres99 As I said above, he should add the part to README because here in issue, no one will find it.

**Comment by Feodor2** [comment]
> he should add the part to README

Sure, but before i want to make certain that it works well.

**Comment by L3P3** [comment]
I sent a symbolic amount for testing.

**Comment by Feodor2** [comment]
Good, it works, bud code changes, now there is another code, must i update it every day?

**Comment by L3P3** [comment]
No, just use the same. You can generate as many codes as you want but all will keep working!

**Comment by noellarkin** [comment]
Same here, I actually don't mind if you make MyPal a paid product - - I want you guys to keep developing this browser, it's the only working option for windows XP right now

**Comment by L3P3** [comment]
@Feodor2 could you please add code to readme and close issue? Please use code I have above, including link to parent project.

**Comment by Feodor2** [comment]
Why to close, we would to look other donate options and confirms about actual donations. Did you donate actually? Was no new money at all.

**Comment by L3P3** [comment]
I test donated, have you got it out? Hope not to donate and you are unable to use it.

I opened and asked for way to donate and you gave me a way so issue is resolved if you make it official and put it to readme + about mypal window. Almost no one will find your dash in an old github issue later.

**Comment by Feodor2** [comment]
Yes i have got and put about it already to the readme.

**Comment by L3P3** [comment]
:ok:

**Comment by L3P3** [comment]
I just donated 80€ but I got frauded, it was wrong address. This is drawback of DASH, it is like CASH. Money gone. :confused:
I sent 80€ again to proper address.

**Comment by Feodor2** [comment]
Uh whats happened, how it was wrong? My address is the same. 
Nevertheless i got the money, thanks.

**Comment by L3P3** [comment]
To get your address from PC to phone, I used DASH QR code generator. It gave me 2 codes, one "donation" and other "your address". Sine it is donation, I used first and code generator page owner got 80€. Very stupid by me, I was too fast.

**Comment by Feodor2** [comment]
Sly bustards, everyone should watch carefully on that.

**Comment by BasBock** [comment]
Love this project, there is just something about Windows XP that makes me very nostalgic and happy.

 Awesome to see that a browser for it is still being developed! I sent a small donation through DASH, did it arrive? 
 
Keep up the good work

**Comment by Feodor2** [comment]
@BasBock thanks it arrived
